packages/ui/src/ui/store/actions/tablet_cell_bundles/index.ts

```diff
@@ -145,7 +145,7 @@ export function fetchTabletsBundles(): TabletsBundlesThunkAction {
                     .get(YTApiId.bundleControllerZones, {
                         parameters: {
                             path: `//sys/bundle_controller/controller/zones`,
-                            attributes: ['tablet_node_sizes', 'rpc_proxy_sizes'],
+                            attributes: ['tablet_node_sizes', 'rpc_proxy_sizes', 'data_centers'],
```

Comment:
You have to use  `//sys/bundle_controller/controller/zones/zone_default` path instead of `//sys/bundle_controller/controller/zones`. So it is better to use `executeBatch` command here. Also we don't need full contend of the attribute it is enough to use `list` subrequest, like:
```bash
$ yt --proxy cluster_id list //sys/bundle_controller/controller/zones/zone_default/@data_centers
dc1
dc1
dc3
```

Reply:
I was wrong, the code works fine.